public/manifest.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "Uptime Kuma",
+    "short_name": "Uptime Kuma",
+    "start_url": "/?source=pwa",
```

We are not using any analytics
```suggestion
    "start_url": "/",
```